spec/instance_title_fields_spec.rb

```diff
@@ -61,10 +61,11 @@
         doc_hash = isd.send(:add_doc_title_fields, {})
         expect(doc_hash[:title_245a_display]).to eq 'foo'
```

Comment:
I realized that lines 57-62 might be better as a separate test (and then can use the shorter version without extra mocking lines as it's a single test), perhaps "removes combination of spaces and trailing punctuation".  Not a big deal but if you're editing the file and you think it's clearer ...